compiler/rustc_ast_lowering/src/lib.rs

```diff
@@ -1033,7 +1031,13 @@ impl<'a, 'hir> LoweringContext<'a, 'hir> {
                     // then to an opaque type).
                     //
                     // FIXME: this is only needed until `impl Trait` is allowed in type aliases.
```

Comment:
Is the comment here still accurate? It looks like before we attempted to support this and now it is just unconditionally an error.